lambdas/src/lib/hrm-ssm-cache/index.ts

```diff
@@ -0,0 +1,63 @@
+import { SSM } from 'aws-sdk';
+
+const ssm = new SSM();
+
+type Config = {
+  values: Record<string, string | undefined>;
+  expiryDate?: Date;
+};
+
+export let config: Config = { values: {} };
+
+export const loadParameters = async ({
+  regex,
+  expiryTime: cacheDuration = 3600000,
```

Comment:
Nitpick but when working with durations, if we don't have the luxury of a nice duration type and are working with primitives, I prefer to include the unit of duration in the parameter names - e.g. `expiryMilliseconds` , `cacheDurationMilliseconds`

Reply:
Good call. Honestly I was banging my head against some typescript/compile problems with the `ms` library and yanked it out so I can move on. Will probably move back to more human friendly args before I'm done.